src/transformers/models/xlm_roberta/tokenization_xlm_roberta.py

```diff
@@ -94,6 +94,17 @@ class XLMRobertaTokenizer(PreTrainedTokenizer):
             modeling. This is the token which the model will try to predict.
         additional_special_tokens (:obj:`List[str]`, `optional`, defaults to :obj:`["<s>NOTUSED", "</s>NOTUSED"]`):
             Additional special tokens used by the tokenizer.
+        sp_model_kwargs (:obj:`dict`, `optional`, defaults to :obj:`None`):
+            Will be passed to the SentencePieceProcessor ``__init__()`` method. Can be used, among other things, to
```

Comment:
```suggestion
            Will be passed to the ``.SentencePieceProcessor.__init__()`` method. Can be used, among other things, to
```
Also, adding a link to a page where we can see all kwargs would be awesome!

Reply:
Documentation of Python API is really bad, I always have to look at the spm code directly:

https://github.com/google/sentencepiece/blob/351600c2971401f4e849147579aa1b5d42f614e1/python/src/sentencepiece/sentencepiece.i#L224

It shows some possible args :)

Reply:
I added a link to the Python API root folder but not to the Python script or code line - since it can change from version to version.